src/ml/data/dwh/setup_dwh.py

```
@@ -53,226 +63,251 @@
-        print("\n" + "="*60)
+        print("\n" + "=" * 60)
         print("DATA INGESTION RESULTS")
-        print("="*60)
+        print("=" * 60)
         print(f"Status: {result['ingestion_status']}")
         print(f"Timestamp: {result['timestamp']}")
-        
-        if result['ingestion_status'] == 'success':
+
+        if result["ingestion_status"] == "success":
             print(f"\nRaw Data Stats:")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)